app/views/decks/index.html.erb

```diff
@@ -42,7 +40,7 @@
           </table>
         </div>
         <div class="pagination">
-          <%= will_paginate @decks, renderer: BootstrapPaginationHelper::LinkRenderer %>
+          <%= will_paginate @decks, renderer: BootstrapPagination::Rails %>
```

Comment:
Is this gem just easier to work with in the long run? If you haven't already, please remove the helper altogether from the repo if you won't be using it